### ISSUE TITLE
timarit.is

### ISSUE THREAD

**Issue by mtwente** [maehr/awesome-digital-history#68]
**Item Details**

Please provide the following details about the item you want to add to the list:

```json
{
	"title": "timarit.is",
	"description": "Collection of newspapers and periodicals of the Faroe Islands, Greenland and Iceland (ca. 1830–today)",
	"url": "https://www.timarit.is",
	"region": ["Arctic"],
	"language": ["is", "fo", "kl", "da"],
	"type": ["newspapers", "magazines", "primary sources"],
	"period": ["contemporary", "modern"]
}
```

Check [maehr.github.io/awesome-digital-history/](https://maehr.github.io/awesome-digital-history/) for the list of available regions, languages, types, and periods.

**Rationale for the addition**

Briefly explain why you think this item should be added to the list and how it fits the criteria for inclusion.

_timarit.is_ provides access to various sources relating to the Atlantic North West to investigate the region's history from perspectives such as politics, colonialism, nation building, literature, business, trade and industry history.

While I think about it: Iceland's national library _Landsbókasafn_ also provides access to [historical maps](https://www.islandskort.is) and [medieval manuscripts](https://www.handrit.is) of Iceland.

Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.

**Comment by maehr** [comment]
Thank you for opening a issue. I like your contribution, even if I can't read any of the languages. Can you open a PR on this issue?

**Comment by mtwente** [comment]
Yes, I was going to, but I was unsure about how useful it actually is in terms of language (even if there are a few German and English sources available). Also:

- Did you already set any standard regarding the _region_ attribute? So far, continents and countries appear to be mixed (and I added something in-between).
- I could also add [islandskort.is](https://www.islandskort.is) and especially [handrit.is](https://www.handrit.is), but I did not want to bloat the list by adding all three separately. These projects are all coordinated by the National Library, which also has  several other sub-projects including sound/music heritage archives. Perhaps it might be best to just link to the [projects overview page](https://landsbokasafn.is/index.php?page=library-websites) once with a long list of keywords, even if these are all separate projects.

**Comment by maehr** [comment]
No, we did not set a standard yet. If you have any ideas (ISO or otherwise), let me know. Ad [islandskort.is](https://www.islandskort.is/) and [handrit.is](https://www.handrit.is/): If these entries are part timartis, your description should probably mention that.
Thanks again

**Comment by maehr** [comment]
> No, we did not set a standard yet. If you have any ideas (ISO or otherwise), let me know. Ad [islandskort.is](https://www.islandskort.is/) and [handrit.is](https://www.handrit.is/): If these entries are part timartis, your description should probably mention that.
> Thanks again

@mtwente Did you figure out which value standard to use for the region field?
